### PR TITLE
Medical Stock Buffs

### PR DESCRIPTION
**Pull request by furrycactus**
- Adds more belts and stethoscopes to sublevel storage, regular ones and EMT ones too to offer choice. If people run out now then I'll be very concerned.
- ~Increases the number of dylovene and norepinephrine bottles in the NanoMed vendors from 4 to 6.~
- ~Increases the number of ATKs and ABKs in the NanoMed vendors from 3 to 5.~
- Takes the EMT/Paramedic hyposprays and puts them on the table in their cave, instead of tucked in both the wall lockers.
- Adds a couple of GPS units to sublevel storage.
- Adds ointment and gauze to the NanoMed vendors.

### CODE REVIEW

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -376,8 +376,8 @@
 	req_access = list(access_medical_equip)
 	vend_id = "meds"
 	products = list(
-		/obj/item/reagent_containers/glass/bottle/antitoxin = 4,
-		/obj/item/reagent_containers/glass/bottle/norepinephrine = 4,
+		/obj/item/reagent_containers/glass/bottle/antitoxin = 6,
+		/obj/item/reagent_containers/glass/bottle/norepinephrine = 6,
```
Comment (Doxxmedearly) [inline code/game/machinery/vending_types.dm]: This is absolutely way too much IMO. There are at least four vendors in medical, and these medications can be made in the pharmacy, or ordered by cargo. Medical is already way too well stocked.
Everything else is fine, though I'm hesitant on the bruise/burn packs.

Comment (furrycactus) [reply code/game/machinery/vending_types.dm]: I disagree on the way too well stocked notion, but I'm not opposed to just keeping it at 4. I'll bump the kits down to 4 each as well.

Comment (furrycactus) [reply code/game/machinery/vending_types.dm]: Nevermind. Putting that back to normal, but adding gauze and ointment to the vendors too.

**Comment by furrycactus** [comment]
!review

**Pull request merged**